addons/mrp/models/mrp_workcenter.py

```diff
@@ -73,6 +73,8 @@ class MrpWorkcenter(models.Model):
         help="Alternative workcenters that can be substituted to this one in order to dispatch production"
     )
     tag_ids = fields.Many2many('mrp.workcenter.tag')
+    capacity_ids = fields.One2many('mrp.workcenter.capacity', 'workcenter_id', string='Product Capacities',
+        help="Specific number of pieces that can be produced in parallel per product.")
```

Comment:
I think you can remove the override of the `copy` method and set `copy=True` on the field
```suggestion
        help="Specific number of pieces that can be produced in parallel per product.", copy=True)
```